docs/PacketFence_Upgrade_Guide.asciidoc

```diff
@@ -1876,6 +1876,11 @@ notify about next certificates expirations for certificates without emails.
 
 == Upgrading from a version prior to 12.0.0
 
+=== Tenant code deprecated
+
+The tenant code use to manage tenants in PacketFence has been deprecated. If you previously used tenants in PacketFence,
```

Comment:
The code used to manage tenants in PacketFence has been removed. If you previously used tenants in PacketFence,